packages/suite/src/components/suite/troubleshooting/TroubleshootingTipsFooter.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button, Row, Text } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { TREZOR_SUPPORT_DEVICE_URL } from '@trezor/urls';
+
+import { Translation } from '../Translation';
+import { TrezorLink } from '../TrezorLink';
+
+export const TroubleshootingTipsFooter = () => (
+    <Row justifyContent="space-between" alignItems="center" margin={{ horizontal: spacings.sm }}>
+        <Text typographyStyle="body">
+            <Translation id="TR_ONBOARDING_TROUBLESHOOTING_FAILED" />
+        </Text>
+
+        <TrezorLink variant="nostyle" href={TREZOR_SUPPORT_DEVICE_URL}>
+            <Button variant="tertiary" size="small">
+                <Translation id="TR_CONTACT_SUPPORT" />
+            </Button>
+        </TrezorLink>
```

Comment:
Please use `Button` with `href` prop + `useExternalLink` hook. It's not a good pattern to wrap `button` with `a`